www/src/components/LayoutHeader/Header.js

```diff
@@ -55,7 +55,7 @@ const Header = ({location}) => (
             width: 'calc(100% / 6)',
           }}
           to="/">
-          <img src={logoSvg} alt="React" height="20" />
+          <img src={logoSvg} alt="" height="20" />
```

Comment:
Why'd the "alt" get removed? Accident?

Reply:
aXe was picking it up that it was a duplicate of the title; which is apparently an anti-pattern. Empty alt tags are used for presentational img tags, so I think we're good 👍

Reply:
> aXe was picking it up that it was a duplicate of the title; which is apparently an anti-pattern

I have never heard this. Interesting.

Upon further consideration, I think images used as links should have alt text describing the destination of the link, not the image itself. So maybe this one should say something like, "React home page"?

Reply:
Oh, I see what you're saying now.